<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QWidget" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>561</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="showResult">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RESULT:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
